.gitattributes

```diff
@@ -34,3 +34,4 @@
 *.woff binary
 
 *.xsd text eol=lf
+*.sh text eol=lf
```

Comment:
@fxprunayre 

Yes that PR does fix the issues and the .gitattributes changes could be backported to 3.12.x

I do have a question regarding the following.

Why was the following used 

`* text eol=lf`

Instead of 

`* text=auto eol=lf`

According to the following

https://github.com/git/git/blob/2e71cbbddd64695d43383c25c7a054ac4ff86882/Documentation/RelNotes/2.10.0.txt#L248-L252

It seems like `* text eol=lf` will declaring all files as text file with  eol=lf.  Which means that a `*.gif` will also be a text file except that `*.gif binary` exists in the .gitattribute to change it back to binary.  So if someone added a new binary file to the system and forgets to add the entry in .gitattribute then it may get corrupts on checkout on certain systems. (At least this is my understanding - I have not tested)

It seems like `* text=auto eol=lf` would only apply the `eol=lf` to files that it detected as text file so it seems like it would not affect any new binary files that are not included in the .gitattribute and would be safer.

Maybe I'm misunderstanding the differences between these values.

Reply:
Looks like you're right https://stackoverflow.com/questions/21472971/what-is-the-purpose-of-text-auto-in-gitattributes-file

Can you check adding `text=auto` and build to see if JS and HTML files formatted by prettier are not modified ? I don't remember if we tested it with @ThmsClement when we checked #6680. For me works fine on linux.